app/router/router.go

```diff
@@ -0,0 +1,116 @@
+package router
+
+import (
+	"regexp"
+	"sort"
+	"strconv"
+
+	"github.com/lbryio/lbrytv/config"
+	"github.com/lbryio/lbrytv/internal/monitor"
+)
+
+type SDKRouter struct {
+	LbrynetServers map[string]string
+	logger         monitor.ModuleLogger
+	servers        []SDK
+}
+
+//SDK represents the Name and Address of an SDK server location
+type SDK struct { //Remove this once we can store names in DB
+	Name    string
+	Address string
+}
+
+func SingleSDKSet(address string) map[string]string {
+	return map[string]string{"default": address}
+}
+
+func New(sdkServers map[string]string) SDKRouter {
```

Comment:
I would cover this `New` function with a unit test. If we move server sorting stuff here, it will make sense to cover in the same test as well.